R/utils.R

```diff
@@ -162,7 +162,7 @@ random_name <- function(prefix = "") {
 rethrow_database_error <- function(msg, call = trace_back()$call[[1]]) {
   tryCatch(
     res <- parse_database_error(msg),
-    error = function(e) cli::cli_abort(msg, call = call)
+    error = function(e) cli::cli_abort("{msg}", call = call)
```

Comment:
My initial guess re: the OP was that this had been the source of the failure (i.e. `parse_database_error()` failed and then the error handling mishandled the message). Turns out it wasn't, but I brought this [in line with cli's documentation](https://cli.r-lib.org/reference/inline-markup.html?q={msg}#escaping-and-) + added a test in the process that felt worth keeping around.